Makefile

```diff
@@ -112,6 +112,9 @@ archive-$(CLI_BINARY).tar.gz:
 	chmod +x $(BINS_OUT_DIR)/$(CLI_BINARY)$(BINARY_EXT)
 	tar czf "$(ARCHIVE_OUT_DIR)/$(CLI_BINARY)_$(GOOS)_$(GOARCH)$(ARCHIVE_EXT)" -C "$(BINS_OUT_DIR)" "$(CLI_BINARY)$(BINARY_EXT)"
 endif
+ifeq ($(GOOS),darwin)
+	cp "$(ARCHIVE_OUT_DIR)/$(CLI_BINARY)_$(GOOS)_$(GOARCH)$(ARCHIVE_EXT)" "$(ARCHIVE_OUT_DIR)/dapr-cli-$(CLI_VERSION).catalina.bottle.tar.gz"
```

Comment:
How about bottle metadata ? Should we need to add metadata DSL in the bottle ?

Reply:
No. That is done in a separate repo. I will create that once we merge this.

Reply:
Which separate repo, will it be checked into this repo as well?

Reply:
It will be a new repo under Dapr. It will be called homebrew-tap. This is as per the instruction from Homebrew. See artursouza/Homebrew-tap as example. It contains a rb file with all metadata needed for Homebrew to install it.

Reply:
Could you explain how the release process would look like?

Reply:
For 0.6, this change will automate the binary upload. The metadata will be manual for 0.6, which means there will be a PR that updates the version in the metadata file after a cli release.

For 0.7, I plan to have an automation to automatically update the metadata after cli release.

Reply:
Sounds good, I have marked the issue as 0.7.0 for now to track the automation as well.